<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="checkBox_check3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>142</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check3</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_check2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>121</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check2</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_check4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>163</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check4</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_check1">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Check1</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Group Box</string>
   </property>
   <widget class="QCheckBox" name="groupBox_checkBox_check3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>71</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="groupBox_checkBox_check1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>29</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="groupBox_checkBox_check4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="groupBox_checkBox_check2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Check2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
